core/src/main/java/org/apache/carbondata/core/scan/filter/executer/ImplicitIncludeFilterExecutorImpl.java

```diff
@@ -77,7 +73,7 @@ public BitSet isScanRequired(byte[][] blockMaxValue, byte[][] blockMinValue,
   }
 
   @Override
-  public void readColumnChunks(RawBlockletColumnChunks rawBlockletColumnChunks) throws IOException {
+  public void readColumnChunks(RawBlockletColumnChunks rawBlockletColumnChunks) {
```

Comment:
Why  didn't remove the exception? like: org.apache.carbondata.core.scan.filter.executer.FilterExecuter#readColumnChunks

Reply:
Because some subclass need to throw IOException, so it can't be removed